### DISCUSSION TITLE
Looking for translators + New Discord Channel

### DISCUSSION THREAD

**Discussion by lukevella** [lukevella/rallly#226] [Announcements]
Hey everyone, I'm looking for volunteer translators who would be willing to help translate the app to other languages. I've written a short intro guide here for anyone that is interested: https://github.com/lukevella/rallly/wiki/Guide-for-translators

I've also created a new [discord server](https://discord.gg/uzg4ZcHbuM) to help with collaboration between translators and myself. It's not just for translators though so please join if you are interested in the project or have any questions. There are channels for getting support, self-hosting and providing feedback. I hope you will [join me](https://discord.gg/m5UFXavc2C) there!
 
Thanks! 👋

**Comment by fnagato** [comment]
Hi Luke!

What do you think about adding the Brazilian Portuguese translation? I can be a volunteer!
I sent you an email just now.

Best regards,
Felippe Nagato.

**Comment by Ant0n3lla** [comment]
Hi Luke!

What do you think about adding the Italian translation? I can be a volunteer!
Best regards
Antonella

**Reply by lukevella** [reply]
Hi @Ant0n3lla, that would be great! Someone already submitted Italian translations this weekend but it would still be great if you can join the project on [Crowdin](https://crwd.in/rallly). There will be new translations needed in the future so if you join the project you will be notified when new translations are needed and you could help proofread existing ones. Thanks!
